### ISSUE TITLE
Illegal callback invocation from native module. This callback type only permits a single invocation from native code.

### ISSUE THREAD

**Issue by JonasWho** [Artirigo/react-native-kontaktio#17]
We are getting this error in Bugsnag. I am not sure wether it has to be resolved by this library or in React Native.  Do you have any idea why this happens?

```
java.lang.RuntimeException ·CallbackImpl.java:32
Illegal callback invocation from native module. This callback type only permits a single invocation from native code.

CallbackImpl.java:32 · com.facebook.react.bridge.CallbackImpl.invoke
PromiseImpl.java:32 · com.facebook.react.bridge.PromiseImpl.resolve
KontaktModule.java:107 · com.artirigo.kontaktio.KontaktModule$1.onServiceReady
InternalProximityManager.java:72 · com.kontakt.sdk.android.ble.manager.internal.InternalProximityManager$1.onServiceConnected
LoadedApk.java:1453 · android.app.LoadedApk$ServiceDispatcher.doConnected
```

<img width="935" alt="screen shot 2017-07-01 at 12 11 16" src="https://user-images.githubusercontent.com/925342/27761191-97157b5c-5e57-11e7-9b8d-e3ecf99900a2.png">

**Comment by Andruschenko** [comment]
Thanks for filing an issue. Could you please provide steps to reproduce the error!

**Comment by JonasWho** [comment]
I've send a PR now, to hopefully fix it.

**Comment by Andruschenko** [comment]
Fixed by merging PR #21
